### DISCUSSION TITLE
[职位开放] 暑期课程与秋期导论课程助教申请

### DISCUSSION THREAD

**Discussion by will-ww** [X-lab2017/open-wonderland#34] [1️⃣ 新鲜事]
我们的暑期与秋季学习有两门课程，每门课程需要两个助教，有兴趣的可以来申请~

**1. 设计思维与精益创业**
- 开课时间：2022年6月27日 ~ 7月22日，周一至周五，每天上午2节课（共18次课）
- 工作内容：线上会议设置、课程项目设计、课程答疑、项目管理与评价
- 仓库地址：https://github.com/X-lab2017/DesignThinking-LeanStartup
- 开放职位：2人
- 要求：对项目管理有一定了解（基于GitHub）

**2. 数据科学与工程导论**
- 开课时间：2022年9月~12月，每周一，2节理论课 + 2节实验课（共18次课）
- 工作内容：实验课答疑、作业批改、期中和期末成绩汇总、基于开源协作日志数据的实验设计
- 仓库地址：https://github.com/X-lab2017/DaSE101
- 开放职位：2人
- 要求：对开源协作日志数据有一定了解（OpenDigger）

直接在帖子下面回复即可占坑~

**Comment by bifenglin** [comment]
申请 数据科学与工程导论。有社会计算课程的一些经验。

**Comment by xgdyp** [comment]
申请 数据科学与工程导论  入学前读过导论的书，对里面的内容比较熟悉

**Comment by tyn1998** [comment]
> 1. 设计思维与精益创业

我我我

**Comment by zhicheng-ning** [comment]
> 1.设计思维与精益创业

申请～

**Reply by will-ww** [reply]
确实也还蛮适合你们两位的，后面我们一起来细化~

**Comment by will-ww** [comment]
两位，我们下学期的导论课程，准备就用语雀文档作为课程的课件方案，我去和语雀团队聊聊能否支持Jupyter Notebook，这样就完美了~

https://xlab2017.yuque.com/docs/share/67c44ebd-2aad-4658-a05b-d67cef31ec83?# 《💡新一代交互式“数字课本”》

@bifenglin @xgdyp

**Reply by bifenglin** [reply]
语雀有回复么？
若语雀不支持代码在线演示的话，可以与github结合方式。

**Reply by will-ww** [reply]
目前还没有呢，GitHub怎么结合？

**Reply by bifenglin** [reply]
直接clone到本地，用本地jupyter演示。
